How can I run webpack in my Camunda modeler?

I have another problem:
Because I didn't manage to  automatically store the bpmns in the project I tried to copy the XML Code which was generated through selection of the test case in the plugin in Camunda. Copied the code into a .bpmn file and stored it in the project (maven-plugin/src/main/ressources). But when I run: "BPMN-driven-testing:generator I get a build failure: 

[INFO] Found BPMN file: orderprocess.bpmn
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.980 s
[INFO] Finished at: 2021-11-25T17:48:34+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.camunda.community:bpmn-driven-testing-maven-plugin:0.3.0:generator (default-cli) on project bpmn-driven-testing-maven-plugin: Unexpected error occurred: SAXException while parsing input stream: Fatal Error: URI=null Line=1: Präfix "bpmn" für Element "bpmn:process" ist nicht gebunden. -> [Help 1]

--- a/README.md
+++ b/README.md
@@ -14,6 +14,22 @@ The extension consists of:
 - [Camunda Modeler plugin](camunda-modeler-plugin) for a visual selection and the management of test cases
 - [Maven plugin](maven-plugin) for generation of JUnit based test code
 
+## Features
+- Visual test case selection
+- Automatic path finding with
+  - Support for embedded sub processes
+  - Support for boundary events
+  - BPMN collaborations with one expanded participant
+  - Loop detection
+- Generated test cases provide
+  - Automatic handling of wait states
+  - Call activity stubbing for isolated testing - see [CallActivityTest](maven-plugin/src/test/it/advanced/src/test/java/org/example/it/CallActivityWithMappingTest.java)
+  - Fluent API to override default behavior
+  - Multi instance activity support - see [tests](maven-plugin/src/test/it/advanced-multi-instance/src/test/java/org/example/it)
+- Spring test support - see [AdvancedTest](maven-plugin/src/test/it/advanced-spring/src/test/java/org/example/it/AdvancedTest.java)
+- Testing of arbitrary paths through a BPMN process
+- Test case validation and migration, when a BPMN process was changed - see [docs](docs/test-case-validation-and-migration.md)
+
 ## How does it work?
 
 ### Select test cases
@@ -52,7 +68,7 @@ When the BPMN model is saved, the selected test cases are attached to the BPMN p
 ```
 
 ### Generate test cases
-To generate the code for the selected test cases, a developer must run the **generator** goal of the [bpmn-driven-testing-maven-plugin](maven-plugin) - in Eclipse select the project and press **ALT+F5** to update.
+To generate the code for the selected test cases, a developer must run the **generator** goal of the [bpmn-driven-testing-maven-plugin](maven-plugin) - in **Eclipse** select the project and press **ALT+F5** to update.
 The goal finds all *.bpmn files under `src/main/resources` and looks for BPMN processes with a `bpmndt:testCases` extension element.
 Each test case will result in a [JUnit test rule](https://github.com/junit-team/junit4/wiki/Rules) - in this example: `generated.order_fulfillment.TC_Happy_Path`.
 
@@ -97,9 +113,9 @@ public class OrderFulfillmentTest {
 
     // enrich and execute test case
     tc.createExecutor()
-      .withBusinessKey("order-123-456")
+      .withBusinessKey("order-20210623-0001")
       .withVariable("customerId", 123)
-      .withVariable("items", "[{\"id\": 1, \"quantity\": 3},{\"id\": 7, \"quantity\": 1}]")
+      .withVariable("customerType", "NEW")
       .withBean("approveOrder", new ApproveOrderDelegate())
       .verify(pi -> {
         // verify state after execution
@@ -114,22 +130,6 @@ When a test is started, the generated test rule handles the creation of the proc
 On the other hand, the test case execution handles the process instance start, applies the specified behavior and verifies that the process instance has passed the correct activities.
 
 A developer can solely focus on the actual implementation!
-
-## Features
-- Visual test case selection
-- Automatic path finding with
-  - Support for embedded sub processes
-  - Support for boundary events
-  - BPMN collaborations with one expanded participant
-  - Loop detection
-- Generated test cases provide
-  - Automatic handling of wait states
-  - Call activity stubbing for isolated testing - see [CallActivityTest](maven-plugin/src/test/it/advanced/src/test/java/org/example/it/CallActivityWithMappingTest.java)
-  - Fluent API to override default behavior
-  - Multi instance activity support - see [tests](maven-plugin/src/test/it/advanced-multi-instance/src/test/java/org/example/it)
-- Spring test support - see [AdvancedTest](maven-plugin/src/test/it/advanced-spring/src/test/java/org/example/it/AdvancedTest.java)
-- Testing of arbitrary paths through a BPMN process
-- Test case validation and migration, when a BPMN process was changed - see [docs](docs/test-case-validation-and-migration.md)
 
 ## More screenshots
 

--- a/camunda-modeler-plugin/README.md
+++ b/camunda-modeler-plugin/README.md
@@ -1,5 +1,7 @@
 # Camunda Modeler Plugin
 
+## Installation
+
 1. Ensure that Camunda Modeler 4.4.0 or higher is installed
 2. [Download](https://github.com/camunda-community-hub/bpmn-driven-testing/releases/latest/download/bpmn-driven-testing-plugin.zip) latest Camunda Modeler plugin release
 3. Unpackage downloaded ZIP file to the `resources/plugins/` directory of the Camunda Modeler installation
@@ -7,35 +9,39 @@
 5. Verify that the plugin is available within the **Plugins** menu as `BPMN Driven Testing` > `Show / Hide` (shortcut CTRL+T).
 
 ## Development
-For the plugin development `npm` (a Node.js installation) is required.
+This and the subsequent sections are only important for plugin development!
 
-1. (Only once) Link the camunda-modeler-plugin directory of the GIT repository as "bpmn-driven-testing" modeler plugin
+For the plugin development a [NodeJS 14.x](https://nodejs.org/download/release/latest-v14.x/) installation is required.
 
-Under Windows, run as administrator
+1. Link the `camunda-modeler-plugin/` directory as Camunda Modeler plugin
 
-```
-cd <camunda-modeler-installation>
-mklink /d resources\plugins\bpmn-driven-testing <repository>\camunda-modeler-plugin
-```
-
-If you want to remove the link:
-
-Under Windows, run as administrator
+Under Windows, run as administrator:
 
 ```
-cd <camunda-modeler-installation>
-rd resources\plugins\bpmn-driven-testing
+mklink /d <camunda-modeler>\resources\plugins\bpmn-driven-testing <repository>\camunda-modeler-plugin
 ```
 
-2. Start the Camunda Modeler
-3. Press **F12** to show developer tools
-4. Run webpack to build `dist/client.js` on the fly, when saving changes
+Under Unix:
+
+```
+ln -s <repository>/camunda-modeler-plugin <camunda-modeler>/resources/plugins/bpmn-driven-testing
+```
+
+2. Install dependencies
+
+```
+npm install
+```
+
+3. Run webpack to build the plugin on the fly, when code changes are saved to file
 
 ```
 npm run dev
 ```
 
-5. To reload the plugin within the Camunda Modeler, press **CTRL+R** in developer tools
+4. Start the Camunda Modeler
+5. In Camunda Modeler, press **F12** to show developer tools
+6. After code changes, press **CTRL+R** in developer tools to reload the plugin within the Camunda Modeler
 
 ## Testing
 Tests are performed using [mocha](https://mochajs.org/) and [chai](https://www.chaijs.com/).

--- a/maven-plugin/README.md
+++ b/maven-plugin/README.md
@@ -3,6 +3,9 @@ The plugin's `generator` goal runs within Maven's default lifecycle in phase `ge
 It generates the test code under `target/bpmndt` and adds this directory as test source directory, which is automatically compiled during the `test-compile` phase.
 The compilation results (test cases and [API classes](src/main/java/org/camunda/community/bpmndt/api)) will be available in the test classpath afterwards.
 
+:warning: With **IntelliJ IDEA**, you may need to [add the additional test source directory manually](https://www.jetbrains.com/help/idea/testing.html#add-test-root).
+Right click on `target/bpmndt` > `Mark Directory as` > `Test Sources Root`
+
 ## Usage
 
 ```xml
@@ -37,9 +40,21 @@ Add dependencies, which are required to execute the generated test code:
 
 ```xml
 <dependency>
+  <groupId>org.camunda.bpm</groupId>
+  <artifactId>camunda-engine</artifactId>
+</dependency>
+
+<!-- Required for plugin version 0.3.0 and lower -->
+<dependency>
+  <groupId>org.camunda.bpm</groupId>
+  <artifactId>camunda-engine-plugin-spin</artifactId>
+</dependency>
+
+<dependency>
   <groupId>junit</groupId>
   <artifactId>junit</artifactId>
   <version>${junit.version}</version>
+  <scope>test</scope>
 </dependency>
 <dependency>
   <groupId>com.h2database</groupId>
@@ -105,6 +120,8 @@ Recommended versions:
 | Spring Framework   | 5.2.8.RELEASE+ |
 
 ## Testing
+This section is only important for plugin development!
+
 Beside unit tests, a set of integration tests exist under [src/test/it](src/test/it).
 These tests verify that the Maven plugin works correctly when executed within a Maven build.
 The integration tests are implemented using the [Robot Framework](https://robotframework.org/) (Java implementation).

